public/locales/en/translation.json

```diff
@@ -147,8 +147,10 @@
         "discordRPC": "Enable Discord Rich Presence",
         "egs-sync": "Sync with Installed Epic Games",
         "enableFSRHack": "Enable FSR Hack (Wine version needs to support it)",
+        "esync": "Enable Esync",
```

Comment:
We cannot edit those files directly otherwise we can have conflicts with the Weblate branch, that deals with all translations. So what you need to do is, after adding the keys on the JSX file (and I saw that you did already) is to run this command:
`yarn i18n`
With that, it will update all language files with the new keys. :)

Reply:
Awesome. done !